lib/get_opts.py

```diff
@@ -20,7 +20,8 @@
 update_parser = subparsers.add_parser('update', help='download updated versions of scripts')
 new_parser = subparsers.add_parser('new', help='create a new sample retriever script')
 new_json_parser = subparsers.add_parser('new_json', help='CLI to create retriever datapackage.json script')
-edit_json_parser = subparsers.add_parser('edit_json', help='CLI to edit retriever datapackage.json script')
+# edit_json_parser = subparsers.add_parser('edit_json', help='CLI to edit retriever datapackage.json script')
```

Comment:
since we have `# edit_json_parser = subparsers.add_parser('edit_json', help='CLI to edit retriever datapackage.json script')` we do not need to comment it out, we should delete it.

same applies to `# edit_json_parser.add_argument('dataset', help='dataset name')`